sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -536,24 +532,13 @@ object ScalaJSPluginInternal {
       }
   )
 
-  val scalaJSTestSettings = (
+  val scalaJSTestSettings: Seq[Setting[_]] = (
       scalaJSTestBuildSettings ++
-      scalaJSRunSettings ++
       scalaJSTestFrameworkSettings ++
       scalaJSTestHtmlSettings
   )
 
-  val scalaJSDefaultBuildConfigs = (
-      inConfig(Compile)(scalaJSConfigSettings) ++ // build settings for Compile
-      inConfig(Test)(scalaJSTestBuildSettings)
```

Comment:
Yes someone had requested that back in the dark days of 0.1-SNAPSHOT or something. At the time I could not yet make a strong enough case against reusing the same class files for JVM and JS, and someone was vocal enough to make me add this abomination. It's time to remove i, though.